platform/ext/target/nordic_nrf/common/nrf_common.cmake

```diff
@@ -95,7 +95,7 @@ endif()
 
 if (NOT DEFINED BUILD_PLAT_TEST)
   message(FATAL_ERROR "Configuration variable BUILD_PLAT_TEST (true|false) is undefined!")
-elseif(BUILD_PLAT_TEST)
+elseif(BUILD_PLAT_TEST OR PSA_API_TEST_NS)
```

Comment:
Instead of doing this @oyvindronningstad I wonder if PSA_API_TEST_NS should just "enable" BUILD_PLAT_TEST